src/main/java/seedu/address/model/event/Event.java

```diff
@@ -21,49 +24,46 @@ public class Event {
     private final Time startTime;
     private final Time endTime;
     private final Description description;
+    private final ObservableList<String> volunteers;
```

Comment:
I might be wrong on this but I had thought we were storing a List<Volunteer> instead of a List<String>. 
If I'm wrong please ignore this comment.